### PR TITLE
notfound completed

### PR DESCRIPTION
**Pull request by fpsolis219**

### CODE REVIEW

File: app/javascript/components/pages/NotFound.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    return (
+        <div> 
+            <h2>Not Found</h2>
+            <img src="https://s3.envato.com/files/198224555/33_ASLAN_404.jpg" alt="" className="" />
+        </div>
+    )
+    }
+    export default NotFound
+
+    // https://media.istockphoto.com/photos/rat-bodybuillders-picture-id92013632?k=20&m=92013632&s=612x612&w=0&h=DvbJ0nztOwXnaKhdNuxDSxugBvUyxkwtU6vLxtHcpPU=
```
Comment (SunkissedQueen) [inline app/javascript/components/pages/NotFound.js]: Please remove any code that is not being used

**Review by SunkissedQueen — DISMISSED** [review]
Please ping me once changes are made

**Comment by fpsolis219** [comment]
we removed the comments and added another test for a p tag element.

**Review by WalkerTexas-coder — APPROVED** [review]
Good work!

**Pull request merged**